### PR TITLE
Modification of previously created pages to provide them with titles #166

### PR DESCRIPTION
**Pull request by monnyy**

### CODE REVIEW

**Comment by kevcamel** [comment]
screen shot 2017-11-08 at 7 08 10 pm

some fancy titles right there ☝️

File: resources/views/postAd.blade.php [hunk 1]
```diff
@@ -1,5 +1,7 @@
 @extends('layouts.app')
 
+@section('title', 'Ad Post')
```
Comment (kevcamel) [inline resources/views/postAd.blade.php]: I think this should be Post Ad and not ~~Ad Post~~, just a small typo

**Review by kevcamel — COMMENTED** [review]
Changes are atomic 👌 Coding style is good as well.

**Comment by BrandonBalala** [comment]
Coding styles are met, clean work.

**Pull request merged**